home/views.py

```diff
@@ -409,20 +407,17 @@ def validateProjectName(request):
         return JsonResponse({"project_name_valid": "Project Name Available"})
 
 
-# Redirecting anonymous login to the right login page
 @login_required(login_url="/login")
 def project_view(request):
     projects = Project_add.objects.all()
     return render(request, "project_view.html", {"projects": projects})
 
 
-# Redirecting anonymous login to the right login page
 @login_required(login_url="/login")
 def profile(request):
     return render(request, "profile.html")
 
 
-# Redirecting anonymous login to the right login page

```

Comment:
@Aryamanz29 Is it really necessary to remove these kinds of comments?

Reply:
It's repeating over every function in views.py that. I think a single comment about login decorators is enough

Reply:
If it's repeated frequently, let it be don't change